Others/Prime.java

```diff
@@ -0,0 +1,75 @@
+package Others;
+
```

Comment:
Please add some tests. For example a main-method with some test-cases or a simple user interface.